test/bin/pyutils/build_bootc_images.py

```diff
@@ -134,6 +149,11 @@ def set_rpm_version_info_vars():
     RHOCP_MINOR_Y1 = ""
     if is_rhocp_available(MINOR_VERSION):
         RHOCP_MINOR_Y = MINOR_VERSION
+    else:
+        rhocp_beta_url = get_rhocp_beta_url_if_available(MINOR_VERSION)
+        if rhocp_beta_url != "":
+            RHOCP_MINOR_Y_BETA = rhocp_beta_url
```

Comment:
I'm not sure we need the var because the default setting is also ""
```suggestion
        RHOCP_MINOR_Y_BETA = get_rhocp_beta_url_if_available(MINOR_VERSION)
```